EnglishTranslation-master/SKILL.tsv

```diff
@@ -4525,42 +4525,42 @@ SKILL_20150714_004524	Increases the damage dealt on an enemy with [Bounce Shot]
 SKILL_20150714_004527	Spiral Arrow: Weakness Attack
-SKILL_20150714_004528	Has a chance to apply Critical to attacks with the effect of [Spiral Arrow] to enemies afflicted with Armor Break. Increases the chance by 10% per attribute level.
+SKILL_20150714_004528	Increases the chance of [Spiral Arrow] inflicting critical damage on enemies afflicted with Armor Break. Increases the chance by 10% per attribute level.
 SKILL_20150714_004529	Increases the damage dealt on an enemy with [Scatter Caltrops] by 1% per attribute level.
 SKILL_20150714_004530	Increases the damage dealt on an enemy with [Stone Shot] by 1% per attribute level.
 SKILL_20150714_004531	Increases the damage dealt on an enemy with [Rapid Fire] by 1% per attribute level.
 SKILL_20150714_004532	Increases the damage dealt on an enemy with [Teardown] by 1% per attribute level.
 SKILL_20150714_004533	Increases the duration of [Scatter Caltrops] by 1 second per attribute level.
-SKILL_20150714_004534	There is a 8% chance per attribute level to  be afflicted with [Bleeding] when stepping on a caltrop.
+SKILL_20150714_004534	There is a 8% chance per attribute level that caltrops will cause [Bleeding].
 SKILL_20150714_004535	Teardown: Teardown Statue
-SKILL_20150714_004536	Allows you to dismantle Dievdirbys' Carve World Tree with [Teardown].
+SKILL_20150714_004536	Allows you to dismantle Dievdirbys' [Carve World Tree] with [Teardown].
 SKILL_20150714_004537	Increases the damage dealt on an enemy with [Broom Trap] by 1% per attribute level.
 SKILL_20150714_004538	Increases the damage dealt on an enemy with [Claymore] by 1% per attribute level.
 SKILL_20150714_004539	Increases the damage dealt on an enemy with [Punji Stake] by 1% per attribute level.
 SKILL_20150714_004540	Increases the damage dealt on an enemy with [Detonate Traps] by 1% per attribute level.
 SKILL_20150714_004541	Increases the damage dealt on an enemy with [Spike Shooter] by 1% per attribute level.
 SKILL_20150714_004542	Increases the damage dealt on an enemy with [Collar Bomb] by 1% per attribute level.
-SKILL_20150714_004543	Tumbleweed of [Broom Trap] will be reduced by 0.2 secs per Attribute level and the number of rotations will decrease by 1 per Attribute level.
+SKILL_20150714_004543	The rotation time of [Broom Trap] will be reduced by 0.2 secs per Attribute level, and the number of rotations will increase by 1 per Attribute level.
 SKILL_20150714_004544	Splash of [Claymore] will increase by 1 per Attribute level.
-SKILL_20150714_004545	As the enemy that was flown to the air by [Punji Stake] falls down on the ground, it will receive damage equal to 50% of Physical ATK and the enemies nearby will also receive damage.
+SKILL_20150714_004545	As the enemy that was sprung in the air by [Punji Stake] falls down on the ground, it will receive damage equal to 50% of Physical ATK and the enemies nearby will also receive damage.
```

Comment:
This is an outdated attribute description. [Click here and check Punji Stake: Knockdown Damage](https://tos.neet.tv/skills/30303).